Exiled.API/Features/Plugin.cs

```diff
@@ -164,12 +165,10 @@ public abstract class Plugin<TConfig, TTranslation> : Plugin<TConfig>
         /// </summary>
         public Plugin()
         {
```

Comment:
You've to call the base constructor as well, the only problem is that by doing this, the `Assembly` property would be overwritten.
So you've to instantiate the name, prefix, author, etc. in this constructor as well.

Reply:
Does the base constructor not run from the class inheriting it? I tried it and got two constructors running, although I didn't test for EXILED.

Reply:
Yes, you're right, the parameterless constructor of inherited classes is being called anyway.